app/helpers/contact_helper.rb

```diff
@@ -0,0 +1,14 @@
+module ContactHelper
+  def contact_link(contact)
+    if contact.present?
+      text = safe_join([fa_icon('envelope'), ' ', contact.name])
+      link_to(text, contact_path(contact))
+    end
+  end
+
+  def contact_from_constant(constant:)
+    contact = Constant.find_by(name: constant).try(:value)
+    contact = Contact.find_by(id: contact)
```

Comment:
There is no association between `Contact` and `Constant`, but yeah, it is two queries :/